integrations/wallets/ledger.md

```diff
@@ -50,6 +50,8 @@ In the next screen, you are prompted to select which hardware wallet you'll like
 
 If MetaMask was able to connect successfully to your Ledger device, you should see a list of five Ethereum-styled accounts. On the contrary, double-check that Ledger Live is closed, you've connected your Ledger device to the computer, unlocked it, and that the Ethereum app is opened. If you are using Chrome, check these [additional steps](#chrome-browser).
 
+### Import Accounts
```

Comment:
Is this necessary? The header it's nestled under is called "Importing your Ledger Account to MetaMask" I feel like this might just be redundant

Reply:
Or maybe it's good to break it up... I don't know 🤷‍♀️